api/HTMLAreaElement.json

```diff
@@ -369,10 +369,10 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "5.1"
+              "version_added": "6"
```

Comment:
This doesn't seem right, http://mdn-bcd-collector.appspot.com/tests/api/HTMLAreaElement/origin fails in Safari 9.1.